tests/back/api/test_runner.py

```diff
@@ -199,8 +199,10 @@ def test_exec_runs(client: TestClient, run_id: int):
     response = client.get(f"/api/v1/run/{run_id}")
     data = response.json()
     assert isinstance(data["train_metrics"], dict)
-    assert isinstance(data["validation_metrics"], dict)
-    assert isinstance(data["test_metrics"], dict)
+    assert "DummyMetric" in data["train_metrics"]
+    assert data["train_metrics"]["DummyMetric"] == 1
+    assert data["train_metrics"] == data["validation_metrics"]
+    assert data["train_metrics"] == data["test_metrics"]
```

Comment:
These shouldn't be equal, we are calculating metrics in different splits, which should have different results

Reply:
But the DummyMetric used in the test always returns 1, so the result is independent of the data.